eng/pipeline/stages/publish-stage.yml

```
@@ -60,6 +66,13 @@ stages:
                 artifact: BuildAssets
               ${{ else }}:
                 artifact: BuildAssetsInternal
+            - ${{ if eq(parameters.publishSymbols, true) }}:
```

Because this param has explicit type `boolean` you can shorten it:

```suggestion
            - ${{ if parameters.publishSymbols }}:
```

(eng/pipeline/stages/builders-to-stages.yml has dynamic types.)